README.md

```diff
@@ -1,24 +1,405 @@
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Experience Exchange is built on a Rails backend leveraging microservices, with a React/TypeScript frontend. The app is deployed via [Heroku](https://dashboard.heroku.com/login) for backend, and [Vercel](https://vercel.com/) for frontend.
 
-Things you may want to cover:
+<br>Link to [Frontend Repo](https://github.com/experience-exchange-2307/fe_experience_exchange)
+<br>Link to [Backend Repo](https://github.com/experience-exchange-2307/be_image_service)
 
-* Ruby version
+## Contributors
+**Backend**
 
-* System dependencies
+- Antoine Aube - [GitHub](https://github.com/Antoine-Aube) || [LinkedIn](https://www.linkedin.com/in/antoineaube/)
+- Ethan Bustamante - [GitHub](https://github.com/ethanb1145) || [LinkedIn](https://www.linkedin.com/in/ethan-bustamante/)
+- Tyler Blackmon - [GitHub](https://github.com/tblackmon-tiel) || [LinkedIn](www.linkedin.com/in/tyler-blackmon/)
 
-* Configuration
+**Frontend**
 
-* Database creation
+- Devin Altobello - [GitHub](https://github.com/daltobello) || [LinkedIn](https://www.linkedin.com/in/devin-altobello-2100036b/)
+- Jen Ziel - [GitHub](https://github.com/jenziel) || [LinkedIn](https://www.linkedin.com/in/jen-ziel400/)
+- Marisa Wyatt - [GitHub](https://github.com/Marisa5280) || [LinkedIn](https://www.linkedin.com/in/marisarwyatt/)
 
-* Database initialization
+## Technologies
+Primary:
+- Ruby 3.2.2
+- Rails 7.0.8
+- PostgreSQL
 
-* How to run the test suite
+Production Gems:
+- [jsonapi-serializer](https://github.com/jsonapi-serializer/jsonapi-serializer)
+- [Faraday](https://lostisland.github.io/faraday/#/)
 
-* Services (job queues, cache servers, search engines, etc.)
+Testing Gems:
+- [rspec-rails](https://github.com/rspec/rspec-rails)
+- [shoulda-matchers](https://github.com/thoughtbot/shoulda-matchers)
+- [FactoryBot](https://github.com/thoughtbot/factory_bot)
+- [Faker](https://github.com/faker-ruby/faker)
 
-* Deployment instructions
+## Installation/Setup
+### Cloning and installing dependencies
+- `git clone <repo_name>`
+- `cd <repo_name>`
+- `bundle install`
+- `rails db:{drop,create,migrate}`
 
-* ...
+### Third Party APIs
+Experience Exchange uses two third party APIs - [Geoapify](https://apidocs.geoapify.com/docs/geocoding/forward-geocoding/#about) for geocoding, and [Unsplash](https://unsplash.com/documentation) for randomized user images. Only Geoapify needs to be setup in the primary repo for geocoding of a user's address.
+
+Follow the Geoapify documentation to get an API key, and place it in the rails credentials file as follows:
+- `EDITOR="code --wait" rails credentials:edit`
+```
+geoapify:
+  api_key: <YOUR_CLIENT_ID>
+```
+
+### Setting up data
```

Comment:
nice I love this section actually. Never though about talking about creation of data in the READme likethis.